docs/static/Devel/Configuring-the-Document-model-of-an-UDV-deployment.md

```diff
@@ -1,305 +0,0 @@
-# WARNING : Outdated document
```

Comment:
Damned for once we had some structured documentation... :-/ Too bad its so updated that fixing it is worthless.